projects/ngx-openlayers/src/lib/sources/tilewmts.component.ts

```diff
@@ -53,7 +54,7 @@ export class SourceTileWMTSComponent extends SourceComponent implements AfterCon
   @Input()
   matrixSet: string;
   @Input()
-  dimensions?: Object;
+  dimensions?: any;
```

Comment:
Can't we also have `{ [key: string]: any }` here?

Reply:
Same here I take the type as defined by openlayers [@type/ol](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/ol/source/WMTS.d.ts#L28)